docs/lob_guide/src/asciidoc/_chapters/work_with_lob.adoc

```diff
@@ -0,0 +1,738 @@
+EMPTY_CLOB()
+```
+
+For more information about examples, see http://trafodion.incubator.apache.org/docs/jdbct4ref_guide/index.html[*Trafodion JDBC Type 4 Programmer’s Reference Guide*].
+
+[#considerations]
+=== Considerations
+
+* The keyword `'APPEND'` can be used to append to lob data that may already exist in a LOB column. If it is not specified, the data is overwritten.
+
+* When a LOB column is updated or overwritten, a new section is written into the LOB data file. The old section remains in HDFS but won’t have any pointer or reference to it.
+
+* When a LOB column is appended, a new chunk is inserted into HDFS and a new row is added in the LOB Descriptor Chunks table to keep track of this new chunk, belonging to the existing row/LOB data. 
+
++
+So a LOB column that contains data that has been appended several times can contain multiple descriptor rows in the Descriptor Chunks table to describe where each chunk belonging to this LOB begins/ends.
+
+* APPEND cannot be used on function `EXTERNALTOLOB`.
+
+* When an external LOB is updated outside of Trafodion, an update needs to be done to update the descriptor files in Trafodion. 
++
+For example, if the external file changes in size, the descriptor file needs to be updated to reflect that. Since this is outside the control of Trafodion, the user needs to do this. If not, when extracting data, only partial data may be returned. 
+
+* If a column is declared with the `STORAGE 'External'` attribute, the `STRINGTOLOB` or `FILETOLOB` functions cannot be used to insert data into that column. 
++
+Similarly, if a column is declared without the `STORAGE 'External'` attribute, the `EXTERNALTOLOB` function cannot be used to insert/update data into that column.
++
+That is, once the storage for a LOB column is declared at CREATE time, the attribute is set and cannot be modified. 
+
+[#select column from a sql table containing lob columns]
+== Select Column from a SQL Table Containing LOB Columns
+
+Selecting a row from a table will give back the lob handle for the lob column. 
+
+NOTE: The entry in each lob column in the SQL table only contains the LOB handle.
+
+Once the LOB handle has been retrieved, it can be used to retrieve the actual lob data from HDFS.
+
+[#syntax]
+=== Syntax
+
+```
+SELECT lob_query-expr[, lob_query-expr] FROM table-name
+```
+
+```
+lob_query_expr is: 
+lobtostring(lob column)      |
+lob column
+```
+
+[#examples]
+=== Examples
+
+* This example selects c2 from table tlob1 and returns the lob handle of c2.
++
+```
+SELECT c2 FROM tlob1;
+C2
+----------------------------------------------------------------------------------------------------
+
+LOBH00000200010423909193650389683319694857010382259683718212310961182290216021"TRAFODION"."SEABASE" 
+
+--- 1 row(s) selected.
+```
+
+[#extract lob data from a sql table containing lob columns]
+== Extract LOB Data from a SQL Table Containing LOB Columns 
+
+Extract lob data can be done in two ways:

```

Comment:
Suggest "Extracting LOB data..."

Reply:
Same as the previous reply: "Using gerund or verb (base) in title/headline is a style issue that should be followed consistently....".

Reply:
I'm planning to change the verb to gerund in all titles/headlines (for consistency) as below:

Working with LOBs
Creating a SQL Table with LOB Columns
Updating a SQL Table Containing LOB Columns
Selecting Column from a SQL Table Containing LOB Columns
Extracting LOB Data from a SQL Table Containing LOB Columns
...

Any comments?

Reply:
I love it :-)

Reply:
Thanks Dave, I've corrected. :)